presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/ColumnStatistics.java

```diff
@@ -31,6 +32,7 @@ public ColumnStatistics(
             DoubleStatistics doubleStatistics,
             StringStatistics stringStatistics,
             DateStatistics dateStatistics,
+            IntegerStatistics timestampStatistics,
```

Comment:
Instead of reusing the `IntegerStatistics` class, I would prefer a new `TimestampStatistics` class to make callers more type safe.